src/views/Game/GameChat.tsx

```diff
@@ -128,8 +128,10 @@ export function GameChat(props: GameChatProperties): JSX.Element {
             debouncedChatUpdate();
         };
 
-        for (const line of goban.chat_log) {
-            onChat(line);
+        for (const log of goban.chat_log) {
+            for (const line of log.lines) {
+                onChat(line);
+            }
```

Comment:
Not sure this change is valid. I get a red screen when game has chat and I exit Zen mode

Reply:
Most definitely wasn't, had some bad typing driving that. Thanks for the catch!